fe/fe-core/src/main/java/org/apache/doris/backup/BackupHandler.java

```diff
@@ -269,10 +271,30 @@ private void backup(Repository repository, Database db, BackupStmt stmt) throws
                     + " is read only");
         }
 
+        // Determine the tables to be backed up
+        Set<String> tableNames = Sets.newHashSet();
+        AbstractBackupTableRefClause abstractBackupTableRefClause = stmt.getAbstractBackupTableRefClause();
+        if (abstractBackupTableRefClause == null) {
+            tableNames = db.getTableNamesWithLock();
+        } else if (abstractBackupTableRefClause != null && abstractBackupTableRefClause.isExclude()) {
+            tableNames = db.getTableNamesWithLock();
+            for (TableRef tableRef : abstractBackupTableRefClause.getTableRefList()) {
+                tableNames.remove(tableRef.getName().getTbl());
+            }
+        }
+        List<TableRef> tblRefs = Lists.newArrayList();
+        if (tableNames.isEmpty()) {
```

Comment:
There may be no table in database at all, and in this case, the `tableNames` is empty, and `abstractBackupTableRefClause` is null. So NPE will thrown.